src/protocols/secure_channel/CASESession.cpp

```diff
@@ -2622,23 +2622,26 @@ System::Clock::Timeout ComputeRoundTripTimeout(ExchangeContext::Timeout serverPr
                                     // Assume peer is idle, as a worst-case assumption (probably true for
                                     // Sigma1, since that will be our initial message on the session, but less
                                     // so for Sigma2).
-                                    System::Clock::kZero, remoteMrpConfig.mActiveThresholdTime) +
+                                    System::Clock::kZero, remoteMrpConfig.mActiveThresholdTime, isFirstMessageOnExchange) +
         serverProcessingTime +
         GetRetransmissionTimeout(localMRPConfig.mActiveRetransTimeout, localMRPConfig.mIdleRetransTimeout,
                                  // Peer will assume we are active, since it's
```

Comment:
I can't suggest on the whole comment here, since it includes removals, but this comment should say:
```
    // Peer will be responding to our message, so isFirstMessageOnExchange should be false
    // and the timestamp does not matter.
```

or something along those lines.

Reply:
updated, thanks